board/safety/safety_toyota.h

```diff
@@ -203,6 +226,19 @@ static bool toyota_tx_hook(const CANPacket_t *to_send) {
       }
     }
 
+    // SecOC cars block any form of LTA actuation for now
+    if (toyota_secoc && (addr == 0x131)) {
```

Comment:
I would now label these three blocks with message names (ideally we make a struct that we can use instead of addresses soon)

Reply:
Do you mean trailing comments, like `// STEERING_LTA_2.LTA_REQUEST`?

Reply:
Added comments, I'm assuming that's what you were looking for if we're deferring an overall refactor of the entire safety to use message naming macros.